src/interfaces/use-query-options.ts

```diff
@@ -0,0 +1,17 @@
+import { CatchResultHandler } from "andculturecode-javascript-core";
+import { ListServiceHook } from "../types/list-service-hook-type";
+import { NestedListServiceHook } from "../types/nested-list-service-hook-type";
+
+export interface UseQueryOptions<
+    TRecord,
+    TQueryParams,
+    TPathParams = undefined
+> {
+    service:

```

Comment:
Nitpick: Alphabetize and per commentary above, rename to better convey that it is a 'hook' by way of its name